### PR TITLE
ci: add pre-commit, molecule, github workflows

### PR DESCRIPTION
**Pull request by juju4**
add corresponding fixes for ansible-lint

### CODE REVIEW

**Comment by Ajnasz** [comment]
Hi @juju4 
Can you explain (in pr description) to me a bit what are the changes here?

Thank you!

File: meta/main.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  standalone: true
+  platforms:
+    - name: EL
+      versions:
+        - all
+    - name: Ubuntu
+      versions:
+        - jammy
+        - focal
+        - bionic
```
Comment (Ajnasz) [inline meta/main.yml:17]: I'm not sure Ubuntu must be listed specifically, while others like Debian is not.

File: molecule/default/requirements.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+
+collections:
+  - ansible.posix
+
+roles:
+  - name: geerlingguy.postgresql
```
Comment (Ajnasz) [inline molecule/default/requirements.yml:7]: In the readme I recommend [ANXS.postgresql](https://github.com/ANXS/postgresql).Can we use it here too? There is no special reason for it, just for consistency.

File: .github/workflows/galaxy.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+        with:
+          path: ajnasz.miniflux
+      - name: galaxy
+        uses: robertdebock/galaxy-action@1.2.1
+        with:
+          galaxy_api_key: ${{ secrets.galaxy_api_key }}
```
Comment (Ajnasz) [inline .github/workflows/galaxy.yml:24]: I guess I'll need to add an api key.

Comment (juju4) [reply .github/workflows/galaxy.yml:24]: Yes. Go to https://galaxy.ansible.com/me/preferences and logged in with github to get API key.

To set in Settings > secrets and variables: Repository secrets.
It also seems there is a new option in Settings > Envvironments: galaxy: Environment secrets.

**Comment by Ajnasz** [comment]
Thank you very much!

**Pull request merged**